pytorch_blade/tests/disc/pdl/test_torchscipt_to_mhlo/test_quantization.py

```diff
@@ -294,10 +488,9 @@ def forward(self, x):
     return %3 : tensor<2x3xf32>
   }
 }
-
         """
         self._test_torchscipte_to_mhlo(traced_model._c, expect_str, self.pdll_files, enable_int8=True)
 
 
 if __name__ == "__main__":
-    unittest.main()
+    unittest.main()
```

Comment:
Although we do not enable python style check in this repo, it is better for each developer to obey the pep8 rules. There should be an empty line in the end of each python file

Reply:
Agree with you, have modified.